hca/default_config.json

```diff
@@ -5,7 +5,7 @@
     "swagger_url": "https://dss.data.humancellatlas.org/v1/swagger.json"
   },
   "DCPQueryClient": {
```

Comment:
key `DCPQueryClient` can be removed from this file

Reply:
👍